crates/ruff_python_formatter/src/string/mod.rs

```diff
@@ -119,42 +119,35 @@ impl StringLikeExtensions for ast::StringLike<'_> {
                             context.source().contains_line_break(literal.range())
                         }
                         ast::FStringElement::Expression(expression) => {
-                            if is_f_string_formatting_enabled(context) {
```

Comment:
The change in this diff block removes the `else` block because the entire function call is behind `is_f_string_formatting_enabled` check.